src/ResourceManager/Sql/Commands.Sql/Services/Util.cs

```diff
@@ -88,11 +88,26 @@ public static bool AreEmailAddressesInCorrectFormat(string emailAddresses, char
             }
 
             string[] emailAddressesArray = emailAddresses.Split(seperator).Where(s => !string.IsNullOrEmpty(s)).ToArray();
+            return AreEmailAddressesInCorrectFormat(emailAddressesArray);
+        }
+
+        /// <summary>
+        /// Checks if email addresses are in a correct format
+        /// </summary>
+        /// <param name="emailAddresses">The email addresses</param>
+        /// <returns>Returns whether the email addresses are in a correct format</returns>
+        public static bool AreEmailAddressesInCorrectFormat(string[] emailAddresses)
+        {
+            if (emailAddresses == null)
+            {
+                return true;
+            }
+
             var emailRegex =
                 new Regex(string.Format("{0}{1}",
                     @"^(?("")("".+?(?<!\\)""@)|(([0-9a-z]((\.(?!\.))|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w])*)(?<=[0-9a-z])@))",
                     @"(?(\[)(\[(\d{1,3}\.){3}\d{1,3}\])|(([0-9a-z][-\w]*[0-9a-z]*\.)+[a-z0-9][\-a-z0-9]{0,22}[a-z0-9]))$"));
-            return !emailAddressesArray.Any(e => !emailRegex.IsMatch(e));
```

Comment:
why is this validation done client-side instead of server-side?

Reply:
It is done in the server side too.
This code is to prevent extra calls to the backend if not necessary...